crates/bevy_sprite/src/render/mod.rs

```diff
@@ -375,14 +375,15 @@ pub fn extract_sprites(
                     .map(|e| (commands.spawn_empty().id(), e)),
             );
         } else {
-            let atlas_rect = sheet.and_then(|s| s.texture_rect(&texture_atlases));
+            let atlas_rect =
+                sheet.and_then(|s| s.texture_rect(&texture_atlases).map(|r| r.as_rect()));
             let rect = match (atlas_rect, sprite.rect) {
                 (None, None) => None,
                 (None, Some(sprite_rect)) => Some(sprite_rect),
-                (Some(atlas_rect), None) => Some(atlas_rect.as_rect()),
+                (Some(atlas_rect), None) => Some(atlas_rect),
                 (Some(atlas_rect), Some(mut sprite_rect)) => {
-                    sprite_rect.min += atlas_rect.min.as_vec2();
-                    sprite_rect.max += atlas_rect.min.as_vec2();
+                    sprite_rect.min += atlas_rect.min;
```

Comment:
This is undoing https://github.com/bevyengine/bevy/pull/11698

Reply:
Actually, this seems more like a drive-by cleanup by someone, so nevermind.

Reply:
Removing this change may make the PR easier to maintain, but feel free to resolve this either way.